package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "caldav-parser",
+  "version": "0.0.0",
+  "description": "",
+  "keywords": [],
+  "main": "dist/caldav-parser.umd.js",
+  "module": "dist/caldav-parser.js",
+  "typings": "types/caldav-parser.d.ts",
+  "files": [
+    "dist"
+  ],
+  "author": "Khaled FERJANI <kferjani@linagora.com>",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:linagora/caldav-parser.git"
+  },
+  "license": "MIT",
+  "engines": {
+    "node": ">=10.0.0"
+  },
+  "scripts": {
+    "lint": "eslint src --ext .ts 'src/**/*.ts' 'test/**/*.ts'",
+    "lint:fix": "npm run lint -- --fix",
+    "prebuild": "rimraf dist",
+    "build": "tsc --module commonjs && rollup -c rollup.config.ts",
+    "start": "rollup -c rollup.config.ts -w",
+    "test": "jest --coverage",
+    "test:watch": "jest --coverage --watch",
+    "test:prod": "npm run lint && npm run test -- --no-cache"
+  },
+  "jest": {
+    "transform": {
+      ".(ts|tsx)": "ts-jest"
+    },
+    "testEnvironment": "node",
+    "testRegex": "(/__tests__/.*|\\.(test|spec))\\.(ts|js)$",
+    "moduleFileExtensions": [
+      "ts",
+      "js"
+    ],
+    "coveragePathIgnorePatterns": [
+      "/node_modules/",
+      "/test/"
+    ],
+    "coverageThreshold": {
+      "global": {
+        "branches": 90,
+        "functions": 95,
+        "lines": 95,
+        "statements": 95
+      }
+    },
+    "collectCoverageFrom": [
+      "src/*.{js,ts}"
+    ]
+  },
+  "devDependencies": {
+    "@rollup/plugin-commonjs": "^17.1.0",
+    "@rollup/plugin-node-resolve": "^11.2.0",
+    "@rollup/plugin-typescript": "^8.2.0",
+    "@types/jest": "^23.3.2",
+    "@typescript-eslint/eslint-plugin": "^4.19.0",
+    "eslint": "^7.22.0",
+    "jest": "^26.6.3",
+    "jest-config": "^26.6.3",
+    "nodemon": "^2.0.7",
```

Comment:
Why `nodemon`?

Reply:
oops, i was testing stuff with it and forgot to remove it